src/_guides/language/effective-dart/design.md

```diff
@@ -1255,6 +1255,23 @@ List<List<Ingredient>> possibleDesserts(Set<Ingredient> pantry) {
 }
 {% endprettify %}
 
+Sometimes the inferred type is not the type you want the variable to have. For
+example, you may intend to assign values of other types later. In that case,
+annotate the variable with the type you want. This guideline only applies when
+the inferred type is the *same* as the type you would annotate.
```

Comment:
Wait, wouldn't that be *different from*, not *the same as*?

Reply:
Oh wait, I think I'm misinterpreting what "guideline" is referring to. But I think other people will misinterpret it too.

Reply:
Yeah, the guideline is telling you when to *omit* the type. I'll just remove that sentence since it's redundant.